tests/fixtures/kamereon/vehicle_data/hvac-status.zoe_50.json

```
@@ -0,0 +1,10 @@
+{
+  "data": {
+    "type": "Car",
+    "id": "VF1AAAAA555777999",
+    "attributes": {
+      "socThreshold": 40,
+      "hvacStatus": "on"
+    }
+  }
+}
```

Please add the missing carriage return here:
```suggestion
}

```